.github/workflows/email-smoke.yml

```
@@ -0,0 +1,32 @@
+      - name: Send test email
+        if: ${{ secrets.SMTP_HOST != '' && secrets.SMTP_USERNAME != '' && secrets.SMTP_PASSWORD != '' && secrets.SMTP_FROM != '' && secrets.SMTP_TO != '' }}
+        uses: dawidd6/action-send-mail@v3
+        with:
+          server_address: ${{ secrets.SMTP_HOST }}
+          server_port: ${{ secrets.SMTP_PORT || 587 }}
+          username: ${{ secrets.SMTP_USERNAME }}
+          password: ${{ secrets.SMTP_PASSWORD }}
+          subject: "Email Smoke Test"
+          to: ${{ secrets.SMTP_TO }}
+          from: ${{ secrets.SMTP_FROM }}
+          content_type: text/html
```

actionlint (1.7.7)

input "content_type" is not defined in action "dawidd6/action-send-mail@v3". available inputs are "attachments", "bcc", "body", "cc", "connection_url", "convert_markdown", "from", "html_body", "ignore_cert", "in_reply_to", "nodemailerdebug", "nodemailerlog", "password", "priority", "reply_to", "secure", "server_address", "server_port", "subject", "to", "username"

(action)